### DISCUSSION TITLE
How to operate more than one tunnel?

### DISCUSSION THREAD

**Discussion by dennysubke** [Radiokot/umbrel-cloudflared#2] [Q&A] [answered]
Unfortunately, there is no way to configure multiple tunnels, although Cloudflare allows this.  You can then direct app 1 (Photoprism:8087) to my1.domain.com and app 2 (Nextcloud:8081) to my2.domain.com and so on. 

This is also possible using the command prompt https://pimylifeup.com/raspberry-pi-cloudflare-tunnel/

**Comment by Radiokot** [comment, marked as answer]
Hi. To do the routing you described, you don't need multiple tunnels. It is enough to have one, and under it, you can configure as many public hostnames as you need using the Cloudflare Zero Trust dashboard:
![image](https://github.com/Radiokot/umbrel-cloudflared/assets/5675681/fceebb82-0533-4cce-b0e4-4376975155b9)

**Reply by dennysubke** [reply]
Is it then also possible to point /umbrel.local directly to umbrel.mydomain.com?

**Reply by Radiokot** [reply]
Yes, if your Umbrel's hostname is `umbrel.local`. Mine is different, for example. You'll be able to log in, see the apps, visit settings and store, but it won't be possible to open any particular app because of port change.

**Reply by Radiokot** [reply]
And you don't need the preceding slash.

**Reply by dennysubke** [reply]
Does this mean that I can no longer forward my already linked Nextcloud app?

**Reply by Radiokot** [reply]
No. It won't open from the Umbrel dashboard. Keep the direct route to nextcloud.

**Reply by dennysubke** [reply]
Thank you very much for your help. Everything now works the way I wanted it to. 👌

**Reply by carlosmassa** [reply]
Is there any way of connecting to umbrel.local using https? Using this method, the connection to apps is through https and secure, but the connection to umbrel.local through umbrel.mydomain.com is through http and unsecure.

![image](https://github.com/user-attachments/assets/3baa5171-06a0-4096-bf82-34633eefdef6)
